stl/inc/format

```diff
@@ -2549,7 +2549,7 @@ _NODISCARD _OutputIt _Write_integral(
     _OutputIt _Out, const _Integral _Value, _Basic_format_specs<_CharT> _Specs, _Lazy_locale _Locale) {
     if (_Specs._Type == 'c') {
         if (!_In_bounds<_CharT>(_Value)) {
-            _Throw_format_error("integral cannot be stored in charT");
+            _Throw_format_error("integral cannot be stored in char/wchar_t");

```

Comment:
What's the impetus for this change? Just personally, I find this _more_ confusing than `charT`; it might be good to do something like

```suggestion
            _Throw_format_error(is_same_v<_CharT, char> ? "integral cannot be stored in char" : "integral cannot be stored in wchar_t");
```

if you want this to avoid mentioning `charT`

Reply:
@CaseyCarter asked for this change, to follow the existing precedent in https://github.com/microsoft/STL/blob/314f65f831f827dbfb05ebcc0f08c496f5ddf64f/stl/inc/format#L1668-L1669 .

I can change them again...